GenderPayGap.WebUI/Controllers/Admin/AdminDatabaseIntegrityChecksController.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.WebUI.Models.Admin;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace GenderPayGap.WebUI.Controllers.Admin
+{
+    [Authorize(Roles = "GPGadmin")]
+    [Route("admin")]
+    public class AdminDatabaseIntegrityChecksController : Controller
+    {
+
+        private readonly IDataRepository dataRepository;
+
+        public AdminDatabaseIntegrityChecksController(
+            IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+
+        [HttpGet("database-integrity-checks")]
+        public async Task<IActionResult> DatabaseIntegrityChecks()
+        {
+            return View("DatabaseIntegrityChecks");
+        }
+
+        [HttpGet("organisations-with-multiple-active-addresses-ajax")]
```

Comment:
It might be nice if the URL was `database-integrity-checks/organisations-with-multiple-active-addresses-ajax`, so it's clear that it's meant to be part of the DB integrity checks

Reply:
Done